src/zoo/owl_zoo_cmd.mli

```diff
@@ -50,3 +50,6 @@ val print_info : unit -> unit
 
 val start_toplevel : unit -> unit
 (** Start the toplevel system tailored for Owl's zoo system. *)
+
+val extend_dir : string -> string
```

Comment:
Do you call this function out of the module? I think it is just  an internal function within the module, if so no need to expose it in the mli file.

Reply:
Yes, this function intended to provided a way to correctly locate the files in the `~/.owl/zoo` directory in a zoo script.